gravitee-apim-console-webui/src/shared/components/gio-selection-list-option/gio-selection-list-option.harness.ts

```diff
@@ -13,29 +13,17 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import { MatButtonHarness } from '@angular/material/button/testing';
 import { MatListOptionHarness, MatSelectionListHarness } from '@angular/material/list/testing';
 import { ComponentHarness } from '@angular/cdk/testing';
 
-export class ApiCreationV4FormSelectionListHarness extends ComponentHarness {
+export class GioSelectionListOptionHarness extends ComponentHarness {
   static hostSelector = 'form';
```

Comment:
should be `.gio-selection-list` no ? 🤔

Reply:
I was matching the file names for the rest of the component... GioSelectionList would work for me. 
Are the other files in the component correctly named?

Reply:
i talk about the `static hostSelector = 'form';` ^^

GioSelectionListOptionHarness is good 👍